### DISCUSSION TITLE
Changing the Start/Stop button style

### DISCUSSION THREAD

**Discussion by liamthompson105** [mebjas/html5-qrcode#666] [Q&A]
Hi,

I've had a look at how to change the style of the button that starts/stops the camera function but drawn up a blank.
I see on the 2 example sites that they are different. How do you go about changing this? Ideally, I'd like the button in Bootstrap format to go with the rest of my site, but at the moment it looks something more akin to Windows 98...

![Screenshot 2023-01-08 020616](https://user-images.githubusercontent.com/67537770/211177484-3dc95472-03c6-476c-a155-2f21f6d7170d.png)

**Comment by tenmillionteeth** [comment]
For anyone in future, you would simply target the element via its CSS selector, and then style it however you wish.

**Reply by Ander2716** [reply]
whats the ids of the elements for torch and sliding (zoom) please?
